src/platform-implementation-js/dom-driver/gmail/views/gmail-nav-item-view.ts

```diff
@@ -708,6 +754,32 @@ export default class GmailNavItemView {
       });
   }
 
+  private _setupSectionElement(): HTMLElement {
```

Comment:
Because of how loose the typing around gmail-nav-item-view is, I'd strongly prefer to not add new non-private members to our interface.